src/runtime/virtcontainers/sandbox.go

```diff
@@ -995,6 +1004,104 @@ func (cw *consoleWatcher) stop() {
 	}
 }
 
+func (s *Sandbox) addSwap(ctx context.Context, id string, size int64) (*config.BlockDrive, error) {
```

Comment:
NIT: `id` change something more details, like `swapID` or `deviceID`.

Reply:
I changed it to swapID.